plugins/com.google.cloud.tools.eclipse.sdk.ui/src/com/google/cloud/tools/eclipse/sdk/ui/preferences/CloudSdkPrompter.java

```diff
@@ -33,6 +33,8 @@
  * found. Must be called from the SWT UI Thread.
  */
 public class CloudSdkPrompter {
+	
+  private static CloudSdk CLOUD_SDK = new CloudSdkProvider().getCloudSdk();
```

Comment:
This shouldn't be static as the user may change the SDK location preference and this instance will become out of date.